bochs/cpu/decoder/disasm.cc

```diff
@@ -472,6 +472,10 @@ char *disasm_regref(char *disbufptr, const bxInstruction_c *i, unsigned src_num,
     disbufptr = dis_sprintf(disbufptr, "dr%d", srcreg);
     break;
 
+  case BX_TREG:
+    disbufptr = dis_sprintf(disbufptr, "tr%d", srcreg);
```

Comment:
For some reason this part doesn't work, so the (gui) debugger doesn't show the instructions correctly.
Can someone help me?

Reply:
encodings above 0xF are not possible, there is assumption of 4 bit
in order to introduce a new encoding need to open a new space

Reply:
How do I do that?